kentico-mvc-environment/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "administrationWebSiteName": {
+      "value": "GEN-UNIQUE"
+    },
+    "sqlServerName": {
```

Comment:
once you add defaultValues, remove the params from here that have defaultValues in the template

Reply:
Removed params of fields that have defaultValues